runtime.md

```diff
@@ -1,5 +1,47 @@
 # Runtime and Lifecycle
 
+## State
+
+The runtime state for a container is persisted on disk so that external tools can consume and act on this information.
+The runtime state is stored in a JSON encoded file.
+It is recommended that this file is stored in a temporary filesystem so that it can be removed on a system reboot.
+On Linux based systems the state information should be stored in `/run/oci/containers`.
+The directory structure for a container is `/run/oci/containers/<containerID>/state.json`.
+By providing a default location that container state is stored external applications can find all containers running on a system.
+
+* **version** (string) Version of the OCI specification used when creating the container.
+* **id** (string) ID is the container's ID.
+* **pid** (int) Pid is the ID of the main process within the container.
```

Comment:
Maybe clarify the host's process namespace?  For example:

> **pid** (int) The ID of the application process in the host's process namespace.

assuming we land something like #107 to get a definition for “application”.  Although this somewhat conflates the initial process and the whole tree descending from that process under application.  And I'm not sure what happens if there are fork/exec calls to split off new trees and the container *isn't* using process namespacing.